src/ElmSharp/ElmSharp/AccessibleUtil.cs

```diff
@@ -51,18 +51,27 @@ public enum ReadingStatus
     public static class AccessibleUtil
     {
 
+
+
+
         static void AtspiSignalCallback(IntPtr data, string say_signal)
         {
             GCHandle gch = GCHandle.FromIntPtr(data);
             TaskCompletionSource<ReadingStatus> tcs = (TaskCompletionSource<ReadingStatus>) gch.Target;
+
             if (say_signal.Equals("ReadingCancelled"))
```

Comment:
The behavior of 'string.Equals(string)' could vary based on the current user's locale settings. Replace this call in 'ElmSharp.Accessible.AccessibleUtil.AtspiSignalCallback(System.IntPtr, string)' with a call to 'string.Equals(string, System.StringComparison)'.